samples/wifi/monitor/README.rst

```diff
@@ -126,6 +126,61 @@ To enable this feature, use the configuration overlay files :file:`overlay-net-c
 When the offline net capture feature is enabled, incoming IEEE 802.11 packets are routed to the offline storage over the net capture tunnel.
 These packets can then be analyzed using Wireshark.
 
+Wireshark decode as IEEE 802.11
```

Comment:
Is the Wireshark program something we should link to in the requirements? And/or Lua programming language?

Reply:
I can add a link https://www.wireshark.org/ and https://wiki.wireshark.org/Lua links

Reply:
Should I just use RST syntax or add them to `links.txt`?

Reply:
I went with `links.txt`